modules/ROOT/pages/index.adoc

```diff
@@ -4,7 +4,13 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-API Community Manager (ACM) overview here.
+MuleSoft enables your company to build APIs, catalog them with Exchange, and present them to software developers inside and outside your company with API Community Manager (ACM).
+
+ACM relies on MuleSoft's Anypoint Exchange as the single source of truth about published APIs, and Salesforce Community Cloud's ability to add customization, branding, and marketing information and to serve different content to different audiences.
+
+With ACM your company can provide its APIs to internal and external developers, treat the APIs as a product and the developers as customers, and provide rich features such as chat, forums, and support integration.
+
+ACM also introduces API Products. An API Product is a group of APIs with marketing context. For example, a payments product might include three APIs for Apple pay, Android pay, and wire transfers.
```

Comment:
I don't think product should be capped (unless it is a product name)

Reply:
Yes, now that you mention it, I think that should be lowercase. I'm not certain so I'll use lowercase for now and make a note to ask.